website/integrations: template: add configuration validation step (#12535)
goauthentik/authentik: website/integrations/template/service.md

1. Write first step here...

2. Continue with steps....

## Configuration verification

> To confirm that authentik is properly configured with Service Name, log out and log back in via authentik. Specify the validation method (e.g., clicking a button), including its location and label for clarity.

Reviewer: ```suggestion
To confirm that authentik is properly configured with <_Service Name_>, log out and log back in via authentik. 

If there are more specific validation methods for the Service (e.g., clicking a button), include these instructions for clarity.
```
